include/openssl/evp.h

```diff
@@ -1398,6 +1398,7 @@ int EVP_PKEY_CTX_set_signature_md(EVP_PKEY_CTX *ctx, const EVP_MD *md);
 # define EVP_PKEY_OP_DECRYPT             (1<<11)
 # define EVP_PKEY_OP_DERIVE              (1<<12)
 
+
```

Comment:
???

Reply:
I added a op and then deleted it - but not the newline